src/GeositeFramework/js/BasemapSelector.js

```diff
@@ -36,11 +45,18 @@
         hideBasemapList(view, e);
     }
 
-    function showBasemapList(view) {
-        view.$el.find('.basemap-selector-list').show();
+    function toggleBasemapList(view) {
+        if (view.model.get('basemap-list-visible')) {
+            view.model.set('basemap-list-visible', false);
+            view.$el.find('.basemap-selector-list').hide();
+        } else {
+            view.model.set('basemap-list-visible', true);
+            view.$el.find('.basemap-selector-list').show();
+        }
     }
 
     function hideBasemapList(view, e) {
+        view.model.set('basemap-list-visible', false);
         view.$el.find('.basemap-selector-list').hide();
         e.stopPropagation();
```

Comment:
Just realize that this line will intercept map clicks from every element now, so going to push a fixup to adjust how it gets called.